themes/default/content/docs/guides/continuous-delivery/pulumi-kubernetes-operator.md

```diff
@@ -1,15 +1,19 @@
 ---
 title: Pulumi Kubernetes Operator
 meta_desc: This page details how to use the Pulumi Kubernetes Operator to manage deploying
-           stacks based on commits to specific Git branches.
+           stacks based on commits in git, Kubernetes objects, or Flux sources.
 menu:
     userguides:
         parent: cont_delivery
         weight: 1
 ---
 
-This page details how to use the [Pulumi Kubernetes Operator](https://github.com/pulumi/pulumi-kubernetes-operator) to manage deploying
-[Stacks][stack] based on commits to specific Git branches.
+This page details how to use the [Pulumi Kubernetes
+Operator](https://github.com/pulumi/pulumi-kubernetes-operator) to manage deploying
+[Stacks][stack]. The Pulumi program for a Stack can come from a [Program resource][], from git, or from a [Flux source][flux-source].
+
+[Program resource]: https://github.com/pulumi/pulumi-kubernetes-operator/blob/master/docs/programs.md
```

Comment:
Are we pointing to GitHub until we can update/docs to cover Program resources?

Reply:
Yes; there's no reference for Program or for Stack in this website.